provider/kv_test.go

```diff
@@ -410,6 +410,14 @@ func TestKVLoadConfig(t *testing.T) {
 					Key:   "traefik/backends/backend.with.dot.too/servers/server.with.dot/weight",
 					Value: []byte("0"),
 				},
+				{
+					Key:   "traefik/backends/backend.with.dot.too/servers/server.with.dot.without.url",
+					Value: []byte(""),
+				},
+				{
+					Key:   "traefik/backends/backend.with.dot.too/servers/server.with.dot.without.url/weight",
+					Value: []byte("0"),
+				},
```

Comment:
Shouldn't we also have added a test where we encounter a "true" error and make sure we filter?